gravitee-apim-console-webui/src/components/alerts/alert/triggers/trigger-api-hc-endpoint-status-changed.component.ts

```diff
@@ -19,28 +19,29 @@ const AlertTriggerApiHealthCheckEndpointStatusChangedComponent: ng.IComponentOpt
     alert: '<',
   },
   template: require('./trigger-api-hc-endpoint-status-changed.html'),
-  /* @ngInject */
-  controller: function () {
-    this.$onInit = () => {
-      // New alert, initialize it with the condition model
-      if (this.alert.id === undefined) {
-        this.alert.conditions = [
-          {
-            type: 'STRING_COMPARE',
-            operator: 'NOT_EQUALS',
-            property: 'status.old',
-            property2: 'status.new',
-          },
-        ];
+  controller: [
```

Comment:
`[` is needed?

Reply:
normally not. I've seen a lot of useless ones but I must have forgotten some.